bigframes/session/executor.py

```diff
@@ -575,20 +572,8 @@ def _validate_result_schema(
 
 
 def generate_head_plan(node: nodes.BigFrameNode, n: int):
-    offsets_id = bigframes.core.guid.generate_guid("offsets_")
-    plan_w_offsets = nodes.PromoteOffsetsNode(
-        node, bigframes.core.identifiers.ColumnId(offsets_id)
-    )
-    predicate = ops.lt_op.as_expr(ex.deref(offsets_id), ex.const(n))
-    plan_w_head = nodes.FilterNode(plan_w_offsets, predicate)
     # Finally, drop the offsets column
```

Comment:
I think this comment is out of date now.

Reply:
removed